lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -172,6 +185,7 @@ bool writePullInformation(const Ufe::Path& ufePulledPath, const MDagPath& path)
     // Store medata on DG node
     auto              ufePathString = Ufe::PathString::string(ufePulledPath);
     MFnDependencyNode depNode(path.node());
+    MStatus           status;
```

Comment:
FYI: the previous version of the code was re-suing a variable that is now inside the lambda.